core/node/eigenda_proxy/src/blob_info.rs

```diff
@@ -35,6 +60,16 @@ impl Encodable for G1Commitment {
     }
 }
 
+impl TryFrom<DisperserG1Commitment> for G1Commitment {
+    type Error = ConversionError;
+    fn try_from(value: DisperserG1Commitment) -> Result<Self, Self::Error> {
+        Ok(Self {
+            x: value.x,
+            y: value.y,
+        })
+    }
+}
```

Comment:
This could be replaced for a simpler `From` implementation:
```rs
impl From<G1Commitment> for DisperserG1Commitment {
    fn from(value: G1Commitment) -> Self {
        Self {
            x: value.x,
            y: value.y,
        }
    }
}
```

Casting from one type to the other in not error prone.